config/environment.js

```diff
@@ -70,10 +70,15 @@ module.exports = function (environment) {
     // here you can enable a production-specific feature
     delete ENV['ember-cli-mirage'];
 
+    ENV.sentry = {
+      dsn: 'https://dafb34e83c054b9289dc733ff336cabc@o242099.ingest.sentry.io/5430576',
```

Comment:
Is there any way to embed this in an environment variable? I'd *really* prefer not to have it in the source code.

Reply:
we could, but it won't make much of a difference since the value will have to be sent to the browser anyway and thus can't be a secret. If we were talking about using Sentry for the backend I'd totally agree though.

if you want to prevent this DSN from being used on other pages you can set "Allowed Domains" on https://sentry.io/settings/rust-lang/projects/crates-io-frontend/ to restrict on which origins the DSN can be used.

Reply:
btw if you look at https://sentry.io/settings/rust-lang/projects/crates-io-frontend/install/javascript-ember/ you can see that this is essentially how Sentry itself recommends to use this :wink:

Reply:
I'm not that worried about leaking the value, but more about being able to change it without tweaking the source code.

Reply:
alright, just keep in mind that you will have to rebuild the frontend so that it can write the new value in the assets.

any suggestion for the env var? `SENTRY_DSN_WEB`?

Reply:
:+1:

Reply:
done